htdocs/install/mysql/tables/llx_adherent_type.sql

```diff
@@ -30,6 +30,7 @@ create table llx_adherent_type
   statut           smallint NOT NULL DEFAULT 0,
   libelle          varchar(50) NOT NULL,
   morphy           varchar(3) NOT NULL,
+  duration         varchar(6) NOT NULL DEFAULT '1y',
```

Comment:
Definition here must match definition into the migration.
I recommend to keep this field optional with no default value (to keep current behaviour when not defined) so:
duration varchar(6) DEFAULT NULL
like into migration

Reply:
ok, change done ;)